docker-compose.yml

```
@@ -6,7 +6,7 @@ services:
         environment:
             FAKE_DNS: 10.77.77.77
             PKCS11_PROXY_SOCKET: tcp://boulder-hsm:5657
-            BOULDER_CONFIG_DIR: test/config
+            BOULDER_CONFIG_DIR: test/config-next
```

I think this diff needs to be reverted

```suggestion
            BOULDER_CONFIG_DIR: test/config
```